modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -198,13 +198,20 @@ config TFM_IMAGE_VERSION_NS
 	  versions of both the secure firmware and non-secure firmware. This sets
 	  the non-secure firmware's version for rollback prevention.
 
+config TFM_BL1
+	bool "Use BL1 immutable bootloader from TFM"
+	default n
+	help
+          TFM provides a reference immutable bootloader, BL1.
+          This config adds BL1 to the build - built via TFM's build system.
```

Comment:
I think some further elaboration on why you would want to enable this is justified here, to help people understand the difference between BL1 and default-on BL2.

Reply:
Added a line about the use case this is useful for.